### PR TITLE
docs: apply some grammar changes

### PR DESCRIPTION
**Pull request by cskaust**
**Goals:**

- resolves #ISSUE-NUMBER
- ...
- ...

- ...
- ...
- [ ] check and update documentation. See [guide](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md#-contributing-documentation) and ask the team.

### CODE REVIEW

File: .github/CODE_REVIEW_GUIDELINES.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
 ## General PR process  
 
 - The scope of PR should be simple, unique and well-defined. PR should not contain unrelated changes
-- Approve PR only if you are sure about the scope
+- The PR will be approved PR only if you are sure about the scope
```
Comment (JohannesMessner) [inline .github/CODE_REVIEW_GUIDELINES.md:14]:
```suggestion
- The PR will be approved only if you are sure about the scope
```

**Comment by JohannesMessner** [comment]
Hi @cskaust, thanks for your contribution!
Note that we cannot merge your PR until your commit messages [match our guidelines](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md#writing-your-commit-message). Otherwise the CI will complain.

**Comment by cskaust** [comment]
Ok,I will keep that in mind.
I am new to contributing and I will improve myself everyday:)

**Comment by JoanFM** [comment]
> Ok,I will keep that in mind. I am new to contributing and I will improve myself everyday:)

Hey @cskaust ,

We need you to fix the commit message right away, otherwise we cannot proceed with the PR

**Review by samsja — APPROVED** [review]

**Comment by alexcg1** [comment]
Hey @cskaust , thanks for your contribution. :clap:  We're currently giving our docs a big overhaul behind the scenes, so I won't move forwards with merging your PR for now.

For your reference, in future we're going to be following (more or less) the [Google Developer Documentation Style Guide](https://developers.google.com/style)

**Pull request closed without merging**